sdk/translation/azure-ai-translation-document/CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.0b1 (Unreleased)
 
 ### Features Added
+- Added SingleDocumentTranslationClient that is used to invoke `document_translate` API.
+- `document_translate` is the method added to submit a single/synchronous document translation request to the Document Translation service.
+- `DocumentTranslateContent` has been added to specify the `document_translate` request content.
+- Added `get_supported_formats` that returns a list of supported documents and glossaries by the Document Translation service.
 
 ### Breaking Changes
-
-### Bugs Fixed
+- Version `v1.0` is not supported
+- Renamed property `FormatType` to `FileFormatType`
```

Comment:
We have renamed FormatType to FileFormatType and made a few other changes too based on the dotnet SDK review feedback. Here are the commit details from TypeSpec: https://github.com/Azure/azure-rest-api-specs/pull/28239/commits/ffd62b7101c9369b19e8b8cd3c7d85d1a9fd747a . 
@kristapratico Please let me know your thoughts.

Reply:
Looks like this SDK exposes that operation differently than the rest api, there are 2 methods, get_support_document_format and get_supported_glossary_format so this model isnt exposed in the SDK. You should remove this entry from the changelog since it's not exposed in the SDK